src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cells-gap;
+
+  width: $calendar-size;
+  padding: $calendar-padding;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    width: $cell-size;
+    height: $cell-size;
+
+    background-color: $cell-bg-color;
+    border: 1px solid $cell-border-color;
+
+    cursor: pointer;
+
+    transition: background-color, transform 0.5s;
+
+    &:before {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      font-size: 31px;
+    }
+
+    @for $day from 1 through 31 {
+      &:nth-child(#{$day})::before {
+        content: "#{$day}";
+      }
+    }
+
+    &:hover {
+      background-color: $cell-hover-bg-color;
+      transform: translateY(-20px);
+    }
+  }
+
+  $days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+  @each $day, $index in $days {
+    &--start-day--#{$day} :first-child {
+      margin-left: $index * ($cell-size + $cells-gap);
+    }
+  }
+
+  @for $day from 28 through 31 {
```

Comment:
Logic is a little bit broken
<img width="1164" alt="Screenshot 2023-09-29 at 15 22 32" src="https://github.com/mate-academy/layout_calendar/assets/117587463/3bc808b3-0002-4d06-9dea-b809cff568e2">